pkg/cmd/release/create/http.go

```diff
@@ -254,3 +278,24 @@ func deleteRelease(httpClient *http.Client, release *shared.Release) error {
 	}
 	return nil
 }
+
+// Given an http.Response, check if the token used in
+// the request is missing the workflow scope.
+func tokenMissingWorkflowScope(resp *http.Response) bool {
+	scopes := resp.Header.Get("X-Oauth-Scopes")
+
+	// Return false when no scopes are present - no scopes in this header
+	// means that the user is probably authenticating with a token type other
+	// than an OAuth token, and we don't know what this token's scopes actually are.
+	if scopes == "" {
+		return false
+	}
+
+	for _, s := range strings.Split(scopes, ",") {
+		if s == "workflow" {
+			return false
+		}
+	}
+
+	return true
```

Comment:
I always struggle when we should do iteration like this versus something simpler like [`slices.Contains()`](https://pkg.go.dev/slices#Contains):

```suggestion

	return slices.Contains(strings.Split(scopes, ","), "workflow")
```

Reply:
Good call, thanks for the refactor 😁